### PR TITLE
Verify that dismissed_notices is an array before using it like one

### PR DESCRIPTION
**Pull request by borkweb**
Related: https://github.com/moderntribe/the-events-calendar/pull/624

See: https://central.tri.be/issues/45005

### CODE REVIEW

**Comment by ggwicz** [comment]
:white_check_mark:

File: src/Tribe/Admin/Notice/Archive_Slug_Conflict.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public function maybe_add_admin_notice() {
 		$this->page = $page;
 		$dimissed_notices = get_user_meta( get_current_user_id(), 'tribe-dismiss-notice' );
 
-		if ( in_array( 'archive-slug-conflict', $dimissed_notices ) ) {
+		if ( is_array( $dismissed_notices ) && in_array( 'archive-slug-conflict', $dimissed_notices ) ) {
```
Comment (jesseeproductions) [inline src/Tribe/Admin/Notice/Archive_Slug_Conflict.php]: We have two different spellings of $dimissed_notices and $dismissed_notices

Comment (borkweb) [inline src/Tribe/Admin/Notice/Archive_Slug_Conflict.php]: Oh, that's a nice catch! I didn't even notice that on the old code.

Comment (ggwicz) [inline src/Tribe/Admin/Notice/Archive_Slug_Conflict.php]: I didn't notice either—thanks so much, Brian

**Pull request merged**